### PR TITLE
Update ro.json

### PR DESCRIPTION
**Pull request by Van4kk**
- grammar corrections
- translation corrections
- added "Meeting" because it was translated wrong to "Reunion"

### CODE REVIEW

**Comment by Van4kk** [comment]
whats wrong ?

File: locales/ro/ro.json [hunk 1]
```diff
@@ -364,9 +364,9 @@
     "Mauritius": "Mauritius",
     "May": "Mai",
     "Mayotte": "Mayotte",
+    "Meeting": "Întalnire",
```
Comment (andrey-helldar) [inline locales/ro/ro.json:367]: No need to add your own words. This package does an automatic comparison of all keys.

If you need to create your own set of translations, you can use the [`template`](https://github.com/Laravel-Lang/translations-template).

File: locales/ro/ro.json [hunk 2]
```diff
@@ -364,9 +364,9 @@
...
     "Mexico": "Mexic",
-    "Micronesia, Federated States Of": "Micronezia",
-    "Micronesia, Federated States of": "Micronesia, Federated States of",
```
Comment (andrey-helldar) [inline locales/ro/ro.json:369]: You have deleted the key. So you don't have to do it.

`Micronesia, Federated States Of` !== `Micronesia, Federated States of`

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by Van4kk** [comment]
aaah my bad then

**Comment by Van4kk** [comment]
all good now ?

File: locales/ro/ro.json [hunk 3]
```diff
@@ -365,8 +365,8 @@
     "May": "Mai",
     "Mayotte": "Mayotte",
     "Mexico": "Mexic",
-    "Micronesia, Federated States Of": "Micronezia",
-    "Micronesia, Federated States of": "Micronesia, Federated States of",
+    "Micronesia, Federated States Of": "Statele Federate ale Microneziei",
+    "Micronesia, Federated States of": "Micronezia",
```
Comment (andrey-helldar) [inline locales/ro/ro.json:369]:
```suggestion
    "Micronesia, Federated States of": "Statele Federate ale Microneziei",
```

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Mulțumim

**Pull request merged**